app/controllers/export/document_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Export::DocumentController < Export::BaseController
+  self.responder = Api::Responder
+
+  def show
+    @document = Document.find_by(id: params[:id])
```

Comment:
I assume this could just be `Document.find(params[:id])` that could save the need for the 404 handling as Rails hopefully covers that for us. Also are we expecting the interface to this to be the Whitehall private id or the content id?

Reply:
I did originally use `find` here, but it raises a `ActiveRecord::RecordNotFound` exception, which Rails does not handle

Reply:
When you say Rails isn't handling it do you mean in the controller tests or when the application is running?

I'd expect that it's raised in controller tests but that it is actually converted to a 404 when the application is running.

Reply:
There's no 404 returned when running the application using `find`, in place of `find_by`.  There's a 500 error with `ActiveRecord::RecordNotFound in Admin::Export::DocumentController#show`.

It appears this way of handling record not found errors is consistent across all the existing API controllers, e.g. https://github.com/alphagov/whitehall/blob/master/app/controllers/api/governments_controller.rb#L16-L20.

In this case, in addition to the 404 response code, you get a JSON response:
```
{
  "_response_info": {
    "status": "not found"
  }
}
```